lib/itamae/plugin/resource/firewalld_service.rb

```diff
@@ -35,9 +34,8 @@ def pre_action
             current.description = service['description'].text
           end
 
-          if service['port']
-            current.protocol = service['port'].attributes['protocol']
-            current.port = service['port'].attributes['port']
+          current.ports = service.collect('port') do |port|
+            [port.attributes['protocol'], port.attributes['port']]
```

Comment:
`attributes.ports` is `Array of String` but `current.ports` is `Array of Array` yet.

Example:

```ruby

firewalld_service 'my-ssh' do
  short       'my-ssh'
  description 'My perfect ssh!!'
  ports       %w(2222/tcp)
end
```

So, first deploy:

```
 INFO :   firewalld_service[my-ssh] ports will change from '[]' to '["2222/tcp"]'
```

And, second:

```
 INFO :   firewalld_service[my-ssh] ports will change from '["[\"tcp\", \"2222\"]"]' to '["2222/tcp"]'
```